indra/ontology/bio/ontology.py

```diff
@@ -634,6 +644,21 @@ def assert_valid_node(self, label):
         except Exception as e:
             logger.warning(e)
 
+    def remove_edges(self, edges_to_remove):
+        initial_edge_count = len(self.edges)
+        for source, target, e_type in edges_to_remove:
+            if (source, target) in self.edges:
+                # Check that the edge type matches
+                if self.edges[source, target]['type'] != e_type:
+                    continue
+
+                # Remove the edge
+                self.remove_edge(source, target)
```

Comment:
It is unlikely to play a big role here but removing edges one by one tends to be slower than using `remove_edges_from` with a list argument. We can leave it like this for simplicity though.